larpandoracontent/LArMonitoring/EventClusterValidationAlgorithm.cc

```diff
@@ -0,0 +1,337 @@
+/**
+ *  @file   larpandoracontent/LArMonitoring/EventClusterValidationAlgorithm.cc
+ *
+ *  @brief  Implementation of the event-level cluster validation.
+ *
+ *  $Log: $
+ */
+
+#include "Pandora/AlgorithmHeaders.h"
+
+#include "larpandoracontent/LArMonitoring/EventClusterValidationAlgorithm.h"
+
+#include "larpandoracontent/LArHelpers/LArClusterHelper.h"
+#include "larpandoracontent/LArHelpers/LArMonitoringHelper.h"
+
+#include <numeric>
+
+using namespace pandora;
+
+namespace lar_content
+{
+
+EventClusterValidationAlgorithm::CaloHitParents::CaloHitParents() :
+    m_pMainMC {nullptr},
```

Comment:
Don't need to change this, but just a note that clang format will remove the space between the name and the opening brace here.